aptos/api/aux-ts/src/graphql/typeDefs/stake.graphql

```diff
@@ -0,0 +1,116 @@
+input StakePoolInput {
+  coinTypeStake: String!
+  coinTypeReward: String!
+}
+
+type StakePool {
+  type: String!
+  coinInfoStake: CoinInfo!
+  coinInfoReward: CoinInfo!
+  authority: String!
+  startTime: Timestamp!
+  endTime: Timestamp!
+  amountStake: Float!
+  rewardRemaining: Float!
+  lastUpdateTime: Timestamp!
+  accRewardPerShare: Float!
+  apr: Float
+  pendingUserReward(owner: Address!, lastAccRewardPerShare: Float): Float!
+  userPosition(owner: Address!): UserPosition!
+  createEvents(start: Int, limit: Int): [CreateStakePoolEvent]!
```

Comment:
let's stick with pool's graphql naming convention and call it `first` `offset` (can translate in sdk land)

(just following convention from https://graphql.org/learn/pagination/#pagination-and-edges)

Reply:
Ok. Why don't we just name it consistently in the sdk as well? It wasn't clear to me how `first` and `offset` were supposed to be mapped since I don't think they're actually used in the pool resolvers: https://github.com/aux-exchange/aux-exchange/blob/a9113e95dbf4bb26fc338d815e0206d51f633de7/aptos/api/aux-ts/src/graphql/resolvers/pool.ts#L27

Reply:
it's called `start` and `limit` in the SDK to match what `AptosClient` calls it

graphql is just a different naming convention

Reply:
Hm, from that docs page doesn't seem like it's actually a naming convention, just what they use for the example. Seems like we should just do whatever is less confusing which would be to use the same naming convention all the way through. I'd prefer to use `start` and `limit` in graphql since that's what goes to Aptos and I find it less confusing.

Reply:
it is convention based on relay spec https://relay.dev/graphql/connections.htm

offset is just a simplification of "after" since we're not using `cursor`

it's just a standard for pagination in graphql. don't mean to be pedantic about naming -- an actual consequence is this way the UI can just add an client-side package which will expect pagination parameters to be named as such